src/widgets/TokenAnnotationWidget.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <table class="selected-tokens">
+    <tr colspan="3" v-if="selectedToken">
+      <a href @click.prevent="onClear">Clear Filter</a>
+    </tr>
+    <tr v-for="token in tokens" :key="token.veRef">
+      <th class="text">{{ token.value }}</th>
+      <td class="parse">{{ token.tag }}</td>
+      <td class="text">{{ token.lemma }}</td>
+    </tr>
+  </table>
+</template>
+
+<script>
+  import gql from 'graphql-tag';
+  import { URN } from '@scaife-viewer/scaife-widgets';
+  import { MODULE_NS, CLEAR_TOKEN } from '@/reader/constants';
+
+  export default {
+    scaifeConfig: {
+      displayName: 'Token Annotations',
+    },
+    methods: {
+      prettyPrint(token) {
```

Comment:
@paltman I know that the "guts" of this widget came from my spike; we can probably drop `prettyPrint` now.